api/models.py

```diff
@@ -233,3 +233,20 @@ def clean(self):
     def save(self, *args, **kwargs):
         self.full_clean()
         super().save(*args, **kwargs)
+
+
+class UserFeedback(models.Model):
+    """ Stores user feedback data """
+    FEEDBACK = "feedback"
+    BUG = "bug"
+    option = (
+        (FEEDBACK, "feedback"),
+        (BUG, "bug"),
+    )
+    reported_by = models.ForeignKey(User, on_delete=models.PROTECT)
+    message = models.CharField(max_length=200, unique=False, null=False)
```

Comment:
isn't unique False by default?

Reply:
It actually is. Thanks for this. Let me change this.